draft-ietf-rats-corim.md

```diff
@@ -771,26 +779,24 @@ The types defined for a group identified are UUID and variable-length opaque byt
 
 ##### Measurements
 
-Measurements can be of a variety of things including software, firmware,
-configuration files, read-only memory, fuses, IO ring configuration, partial
-reconfiguration regions, etc. Measurements comprise raw values, digests, or
-status information.
+A measurement can be directly compared with attestation evidence, or it can be a quality that is not directly comparable with evidence.
```

Comment:
I cannot really follow "[Evidence] can be a quality that is not directly comparable with evidence". Could you maybe provide an example so it becomes more clear?

Reply:
In general, I think that Evidence is composed of "Measurements" and there might be a bigger disconnect here that I cannot follow.

Reply:
I don't think that an assigned name or serial number is always provided by an attesting environment's measurement mechanism. That information is added metadata provided the reference values are as expected.

pseudocode:

```
reference-values-triple [E, M[integrity-registers: X]]
conditional-endorsement-triple [
  conds: [E, [integrity-registers: X authorities: [me]]]
  [E, M'[name: "Product: Enterprise Edition", svn: 5]]
]
```

I'm not really sure how to express associated metadata in a less-clunky way, unless this conditionality is what is the expected semantics of the non-conditional endorsement-triple. It's unclear from the text https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/179#issuecomment-2184116580